Autopilot/AttitudeManager/nbproject/configurations.xml

```diff
@@ -193,8 +196,9 @@
         <property key="use-cci" value="false"/>
         <property key="use-iar" value="false"/>
       </C30>
-      <C30-AS>
-      </C30-AS>
+      <C30-AR>
+        <property key="additional-options-chop-files" value="false"/>
```

Comment:
Can we not commit these config files? Maybe add the project config folder to the git ignore for now. They still need to be within the repo but theres no pointing in updating them any further.